### PR TITLE
[FIX] Fix model properties

### PR DESCRIPTION
**Pull request by hughesbm**
* Change Person emails property to a list of strings
* Move old emails property (list of email objects) back to Customer
* Add customer_person_type property to Person
* Add threads property to Conversation
* Add Thread action_type choice

### CODE REVIEW

**Comment by lasley** [comment]
Make sure to add yourself as a contributor in the readme

**Comment by hughesbm** [comment]
Resolved merge conflict

File: helpscout/models/person.py [hunk 1]
```diff
@@ -53,3 +59,11 @@ class Person(BaseModel):
     modified_at = properties.DateTime(
         'UTC time when this customer was modified.',
     )
+
+    @properties.Bool('customer boolean')
+    def customer_person_type(self):
+        return self.type == 'customer'
```
Comment (lasley) [inline helpscout/models/person.py]: Add a test for this

File: helpscout/models/person.py [hunk 2]
```diff
@@ -53,3 +59,11 @@ class Person(BaseModel):
...
+
+    @customer_person_type.setter
+    def customer_person_type(self, value):
+        self.type = 'customer' if value else 'user'
```
Comment (lasley) [inline helpscout/models/person.py]: And this

**Pull request merged**